essential-projects/datasource_adapter_proxy
# Feature/fix build errors

## What did you change?

This fixes some linting, aswell as some build errors that popped up with the latest popsicle release.

## How can others test the changes?

These are internal changes, related only to the TS build, so everything should run as before.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
